surveyor/collector_statz.go

```diff
@@ -1096,7 +1062,7 @@ func (sc *StatzCollector) Collect(ch chan<- prometheus.Metric) {
 					metrics.newCounterMetric(sc.descs.accBytesSent, float64(as.Data.Sent.Bytes), serverAndAccLabels)
 					metrics.newCounterMetric(sc.descs.accBytesRecv, float64(as.Data.Received.Bytes), serverAndAccLabels)
 					metrics.newCounterMetric(sc.descs.accMsgsSent, float64(as.Data.Sent.Msgs), serverAndAccLabels)
-					metrics.newCounterMetric(sc.descs.accMsgsRecv, float64(as.Data.Sent.Bytes), serverAndAccLabels)
+					metrics.newCounterMetric(sc.descs.accMsgsRecv, float64(as.Data.Received.Msgs), serverAndAccLabels)
```

Comment:
I don't know why GitHub is showing this diff, this is already how it is in `main`: https://github.com/nats-io/nats-surveyor/blob/75cb45736f8a4158c6109852da043af13899303b/surveyor/collector_statz.go#L1097